src/openms/source/ANALYSIS/MAPMATCHING/QTClusterFinder.cpp

```diff
@@ -496,7 +508,8 @@ namespace OpenMS
 
 #ifdef DEBUG_QTCLUSTERFINDER
     OpenMSBoost::unordered_map<Size, OpenMS::GridFeature*> elements;
-    cluster.getElements(elements);
+    // this shouldn't be done anymore
```

Comment:
why?

Reply:
Forgot when and why I did this. Now it should be correct and work.